genie/consortium_to_public.py

```diff
@@ -147,7 +144,19 @@ def consortiumToPublic(
         query_string=f"SELECT * FROM {clinical_tier_release_scope_synid} where releaseScope = 'public'",
     )
 
+    # check if SAMPLE_CLASS is present
+    if not process_functions.check_values_in_column(
```

Comment:
These are great post processing checkers, eventually these will most likely be superceded with the new data quality framework rixing may want to put together.